src/test/java/com/redhat/rhjmc/containerjfr/commands/internal/AbstractConnectedCommandTest.java

```diff
@@ -141,8 +141,8 @@ public String getName() {
         public void execute(String[] args) {}
 
         @Override
-        public boolean validate(String[] args) {
-            return true;
+        public void validate(String[] args) throws FailedValidationException {
+            return;
```

Comment:
The `return` is implicit for a `void` method, so you can just have an empty method body between the curly braces.